mdms/mdms-java/src/main/java/de/hpi/isg/mdms/java/fk/Instance.java

```diff
@@ -0,0 +1,65 @@
+package de.hpi.isg.mdms.java.fk;
+
+import it.unimi.dsi.fastutil.objects.Object2ObjectArrayMap;
+
+import java.util.Map;
+
+/**
+ * Represent an instance in the dataset.
```

Comment:
Yes, i try to encapsulate each IND in the dataset as an Instance. So each FK candidate should also be an Instance